pallets/automation-time/src/lib.rs

```diff
@@ -245,6 +285,49 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		/// Schedule a task to fire an event with a custom message.
+		///
+		/// Before the task can be scheduled the task must past validation checks.
+		/// * The transaction is signed
+		/// * The provided_id's length > 0
+		/// * The time is valid
+		/// * 
+		///
+		/// # Parameters
+		/// * `provided_id`: An id provided by the user. This id must be unique for the user.
+		/// * `time`: The unix standard time in seconds for when the task should run.
+		/// * `message`: The message you want the event to have.
+		///
+		/// # Errors
+		/// * `InvalidTime`: Time must end in a whole minute.
+		/// * `PastTime`: Time must be in the future.
+		/// * `DuplicateTask`: There can be no duplicate tasks.
+		/// * `TimeSlotFull`: Time slot is full. No more tasks can be scheduled for this time.
+		/// * `InvalidAmount`: Amount has to be larger than 0.1 OAK.
+		/// * `InvalidRecipient`: Recipient must be a valid address.
+		/// * `TransferToSelf`: Sender cannot transfer money to self.
+		#[pallet::weight(<T as Config>::WeightInfo::schedule_transfer_task_existing_slot())]
+		pub fn schedule_transfer_task(
+			origin: OriginFor<T>,
+			provided_id: Vec<u8>,
+			time: UnixTime,
+			recipient_id: T:: AccountId,
+			amount: T::Balance,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			let recipientExists = account_exists(recipient_id);
```

Comment:
i found `account_exists` in the same file as `ensure_signed` in the substrate repo: https://github.com/paritytech/substrate/blob/master/frame/system/src/lib.rs#L948

i'm not sure how `ensure_signed` is getting imported in this file. it looks like it's being set as publicly usable for the crate. `account_exists` is a public function, but does not look like it's available to public use in the crate. so i'm curious if you know a better way to do get if an account exists

Reply:
RESOLVED: no `account_exists` because accounts without EXISTENTIAL_DEPOSIT will not exist. we want people to be able to send to new accounts